src/main/java/businessLayer/userTypes/SystemController.java

```diff
@@ -20,7 +20,6 @@
 import businessLayer.userTypes.Administration.*;
 import businessLayer.userTypes.viewers.*;
 import dataLayer.DemoDB;
```

Comment:
Not approved.
Every function that you have written in this class you need to put in a comment.
@alongolo @idoEfron